cypress/support/commands.js

```diff
@@ -451,11 +451,33 @@ Cypress.Commands.add('fillInResourceType', (type, subtype) => {
   cy.get(`[data-testid=publication-instance-type]`).click();
   cy.get(`[data-testid=publication-instance-type-${subtype.replaceAll(' ', '-')}]`).click();
   Object.keys(resourceTypes[type][subtype]).forEach((key) => {
-    const field = resourceTypes[type][subtype][key];
-    fillInField(field);
+    if(key !== 'contributorType') {
+      const field = resourceTypes[type][subtype][key];
+      fillInField(field);
+    }
   });
+  if('contributorType' in resourceTypes[type]) {
+    cy.get(`[data-testid=${dataTestId.registrationWizard.stepper.contributorsStepButton}]`).click();
+    fillInField(resourceTypes[type]['contributorType']);
+}
 });
 
+Cypress.Commands.add('fillInContributors', (type, subtype) => {
+  cy.get(`[data-testid=${dataTestId.registrationWizard.stepper.contributorsStepButton}]`).click();
+  let fields = {};
+  if(type in contributors) {
+    fields = contributors[type];
+  } else if(subtype in contributors) {
```

Comment:
ser ut som prettier ikke har gjort jobben sin? gjelder sikkert flere plasser i så fall
```suggestion
  if (type in contributors) {
    fields = contributors[type];
  } else if (subtype in contributors) {
```